src/main/java/org/sagebionetworks/bridge/services/ActivityEventService.java

```diff
@@ -147,9 +165,58 @@ public void publishActivityEvent(ActivityEvent event) {
      */
     public Map<String, DateTime> getActivityEventMap(String healthCode) {
         checkNotNull(healthCode);
+        
         return activityEventDao.getActivityEventMap(healthCode);
     }
 
+    /**
+     * Gets the activity events times for a specific user in order to schedule against them.
+     */
+    public Map<String, DateTime> getActivityEventMap(Account account) {
+        checkNotNull(account.getHealthCode());
+        Map<String, DateTime> activityMap = activityEventDao.getActivityEventMap(account.getHealthCode());
+        Builder<String, DateTime> builder = ImmutableMap.<String, DateTime>builder();
+        
+        DateTime studyStartDateTime = activityMap.get(ActivityEventObjectType.STUDY_START_DATE.name().toLowerCase());
```

Comment:
Is it worth loading study_start_date from the activities map? This is likely to change, and it's not that expensive to re-compute. Also note that this activity event isn't actually being saved anywhere.